App/Navigation/PhotosNavigation.js

```diff
@@ -19,11 +19,17 @@ import styles, {headerTintColor} from './Styles/NavigationStyles'
 
 const PhotosNav = StackNavigator(
   {
-    TextilePhotos: TextilePhotos
+    TextilePhotos: {
+      screen: TextilePhotos,
+      navigationOptions: {
+        headerTitle: <Image source={require('../Images/TextileHeader.png')} />
+      }
+    }
   },
   {
     // Default config for all screens
     headerMode: 'float',
+    initialRouteParams: { thread: 'default', sharable: true },
```

Comment:
Using the same screen to display default and beta, just passing in different params.